src/style/fims/_base.scss

```diff
@@ -170,4 +170,58 @@ md-checkbox.small {
 .form-horizontal .form-group {
     margin-left: 0px;
     margin-right: 0px;
-}
\ No newline at end of file
+}
+
+
+
+
+
+
+
+//about page/splashpage
+.splash-page {
+position: relative;
+width: 1200px;
+height: 550px;
+margin-bottom: 100px;
+}
+
+//Icon links
+
+.icon-links {
+font-size: 25px;
+font-weight: bold;
+position:absolute;
+bottom: 240px;
+}
+
+.icon-links > img {
+height: 150px;
+position:absolute;
+bottom: 45px;
+}
+
+#about-icon {
+left:100px;
+}
+
+#about-icon > img {
+height: 145px;
+left: -29px;
+}
+ 
+#query-icon {
+left:500px;
+}
+
+#query-icon > img {
+left: -40px;
+}
+
+#workbench-icon {
+left:870px;
```

Comment:
in general it is much better to use relative positioning when possible. Here it looks like you are using absolute positioning, but when you view the site on a larger screen, it will look strange.

I'll view these changes locally and give a suggestion, but from the looks of it, you added 3 "cards". You could put them all in a parent container that centers the children, and set a max-width on the children if you don't want the cards to grow to wide on large displays. Then you can use margin & padding to separate the cards for each other

Reply:
okay, I would suggest using some type of card as I think it would look better. Take a look at https://material.angularjs.org/latest/demo/card for examples.

Also, for the layout, try and make your window smaller, you end up having to scroll to see these things. You can set the # of columns at a screen size so it works better. Ex. for xs screens there should be 1 column, anything bigger 3 columns. Each link would be 1 column. See [angularMaterial docs](https://material.angularjs.org/latest/layout/introduction) for more help of layouts & examples.